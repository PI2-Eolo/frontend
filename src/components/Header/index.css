.header-container {
    width: 100%;
    background-color: #fff;
    display: flex;
    flex-direction: row;
    height: 4em;
    align-items: center;
    padding: 1em;
    border-bottom: 2px solid #EEECFA;
}

h1 {
    display: flex;
    font-family: 'Poppins Thin';
    font-weight: 100;
    font-size: 2em;
}

img {
    display: flex;
    height: 3em;
    padding-right: 1em;
}

@font-face {
    font-family: 'Poppins Thin';
    src: local('Poppins Thin'), url(../../assets/fonts/Poppins-Thin.ttf) format('ttf');
}