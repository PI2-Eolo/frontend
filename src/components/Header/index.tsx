import './index.css';

const Header = () => {
    return (<div className="container">
        <h1 className='title'> header</h1>
    </div>)
};

export default Header;