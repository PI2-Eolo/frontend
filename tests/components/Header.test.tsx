import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../../src/components/Header';

describe('Header component test', () => {
    it('component snapshot', () => {
        const component = renderer.create(
            <Header />,
        );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
})