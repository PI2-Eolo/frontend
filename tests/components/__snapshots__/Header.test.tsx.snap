// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component test component snapshot 1`] = `
<div
  className="header-container"
>
  <img
    alt="Logo"
    src="test-file-stub"
  />
  <h1>
     Dashboard 
  </h1>
</div>
`;
